package.json

```diff
@@ -15,6 +15,7 @@
     "start": "node lib/index.js",
     "start:watch": "nodemon lib/index.js --log",
     "test": "jest --config jest.config.js",
+    "install:ci": "npm ci",
```

Comment:
Can you order this command alphabetically?

Reply:
Sure thing! Commited ^_^